samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -234,11 +238,24 @@ static const char link_rsrp_usage_str[] =
 	"  -u, --unsubscribe,  Unsubscribe for RSRP info\n";
 
 static const char link_ncellmeas_usage_str[] =
-	"Usage: link ncellmeas --single | --continuous |--cancel\n"
+	"Usage: link ncellmeas --single | --continuous |--cancel [<search_type>]\n"
```

Comment:
At least to me this notation is a bit confusing, it looks as if `search_type` would be an option for `--cancel`. Also a space is missing before `--cancel`. How about this?

```suggestion
	"Usage: link ncellmeas --single | --continuous | --cancel | [<search_type>]\n"
```

Reply:
yes, that is how it should be.